services/resetPassword.js

```diff
@@ -0,0 +1,42 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+const userRepository = require('../repositories/user');
+const errors = require('../configuration/errors');
+
+async function showResetPasswordForm(req, res) {
```

Comment:
why we need this. we have frontend for displaying for. All you need is function that will take new password and token, verify user and update password. we don't need this on backend part.